### PR TITLE
Change database engine to postgres

### LINKED ISSUE #9: Change database engine to postgres
# REQUIREMENTS
Change database engine from sqlite3 to postgres inside "database.yml" file

### PR DESCRIPTION
**Pull request by AlejandroCuriel**
# Description
- What?: We change the database engine from sqlite3 to postgres 
- Why?: To start creating the employee, branch and attendance tables

 # Testing
No testing required.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,3 +33,5 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+# Ignore database
+/config/database.yml
```
Comment (JuanequeX) [inline .gitignore]: Add a blank line here

File: config/database.yml [hunk 2]
```diff
@@ -1,25 +1,86 @@
...
+#
+# Read https://guides.rubyonrails.org/configuring.html#configuring-a-database
+# for a full overview on how database connection configuration can be specified.
+#
 production:
   <<: *default
-  database: db/production.sqlite3
+  database: kata_07_reloj_checador_k7_f22_t2_production
+  username: kata_07_reloj_checador_k7_f22_t2
+  password: <%= ENV["KATA_07_RELOJ_CHECADOR_K7_F22_T2_DATABASE_PASSWORD"] %>
```
Comment (JuanequeX) [inline config/database.yml:86]: this file should will be added in the .gitignore file

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (AlejandroCuriel) [reply .gitignore]: Done

File: config/database.yml (hunk 2 above)
Comment (AlejandroCuriel) [reply config/database.yml:86]: Done

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by DerekAyala — APPROVED** [review]
Looks Good

**Review by germanruzca — APPROVED** [review]
LGTM

**Review by nallely64 — APPROVED** [review]
LGTM

**Review by Javi1499 — APPROVED** [review]
Good job!

**Review by YilKanda — APPROVED** [review]

**Pull request merged**